src/components/layouts/PageLayout.tsx

```diff
@@ -7,7 +7,7 @@ type PageLayoutProps = {
 
 export function PageLayout({ children }: PageLayoutProps) {
 	return (
-		<div className="flex flex-col min-h-screen">
+		<div className="flex flex-col min-h-screen font-roboto">
```

Comment:
Lehet inkább jobb lenne a `styles.css`-be tenni egy rule-t a `html`-re, mint egy 'random' React komponensben beállítani a site-wide configot.